packages/vite/src/node/plugins/importAnalysisBuild.ts

```diff
@@ -26,6 +26,8 @@ type FileDep = {
   runtime: boolean
 }
 
+type VitePreloadErrorEvent = Event & { payload?: Error }
```

Comment:
```suggestion
type VitePreloadErrorEvent = Event & { payload: Error }
```

Shouldn't `payload` be required?

Reply:
Ok, updated.